src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizard.css

```diff
@@ -55,5 +55,17 @@
 }
 /* hide rogue label spacing on lightning:select when variant="label-hidden" */
 .THIS .label-hidden > label {
-    display: none;
+    position: absolute!important;
+}
+/*Workaround since label-hidden variant on lightning:inputField is inconsistent in hiding labels for types of fields.
+Revist this Summer '19 and after to see if it's still necessary or the CSS has changed*/
+.THIS .hideLabelFix lightning-grouped-combobox .slds-form-element__label {
+    /*partial copy from .slds-assistive-text*/
+    position: absolute!important;
```

Comment:
I'm assuming there's no alternative to "!important" here?

Reply:
That's a good question; I really just copied/pasted it over and didn't check, but would assume so! This might be specific enough to override without the !important, but would need to test that again.